### PR TITLE
Update Frame-VAD doc

### PR DESCRIPTION
**Pull request by stevehuang52**
# What does this PR do ?

Update doc for Frame-VAD

**Collection**: [ASR]

### CODE REVIEW

**Review by fayejf — DISMISSED** [review]
LGTM. Thanks. Some small comments

File: examples/asr/speech_classification/README.md [hunk 1]
```diff
@@ -1,25 +1,88 @@
 # Speech Classification
 
-This directory contains example scripts to train speech classification and voice activity detection models. 
+This directory contains example scripts to train speech classification and voice activity detection models. There are two types of VAD models: Frame-VAD and Segment-VAD.
 
-# Model execution overview
+## Frame-VAD
 
-The training scripts in this directory execute in the following order. When preparing your own training-from-scratch / fine-tuning scripts, please follow this order for correct training/inference.
+The frame-level VAD model predicts for each frame of the auio whether it has speech or not. For example, with the default config file (`../conf/marblenet/marblenet_3x2x64_20ms.yaml`), the model provides a probability for each frame of 20ms length.
```
[the remark this answers is not in the document]
Comment (stevehuang52) [reply examples/asr/speech_classification/README.md:7]: fixed

File: examples/asr/speech_classification/README.md [hunk 2]
```diff
@@ -1,25 +1,88 @@
...
+    trainer.max_epochs=100
+```
+
+The input manifest must be a manifest json file, where each line is a Python dictionary. The fields ["audio_filepath", "offset", "duration",  "label"] are required. An example of a manifest file is:
+```
+{"audio_filepath": "/path/to/audio_file1", "offset": 0, "duration": 10000,  "label": "0 1 0 0 1"}
+{"audio_filepath": "/path/to/audio_file2", "offset": 0, "duration": 10000,  "label": "0 0 0 1 1 1 1 0 0"}
+```
+For example, if you have a 1s audio file, you'll need to have 50 frame labels in the manifest entry like "0 0 0 0 1 1 0 1 .... 0 1".
+However, shorter label strings are also supported for smaller file sizes. For example, you can prepare the `label` in 40ms frame, and the model will properly repeat the label for each 20ms frame.
```
[the remark this answers is not in the document]
Comment (stevehuang52) [reply examples/asr/speech_classification/README.md:28]: yes

File: tutorials/asr/Offline_ASR_with_VAD_for_CTC_models.ipynb [hunk 3]
```diff
@@ -313,13 +326,15 @@
...
         {
+            "attachments": {},
             "cell_type": "markdown",
             "metadata": {},
             "source": [
                 "# Evaluate the performance of offline VAD with ASR "
             ]
         },
         {
+            "attachments": {},
```
[the remark this answers is not in the document]
Comment (stevehuang52) [reply tutorials/asr/Offline_ASR_with_VAD_for_CTC_models.ipynb:337]: I don't know...it's automatically generated

**Review by fayejf — APPROVED** [review]
LGTM Thanks!

**Pull request merged**